src/core/inputs/udp-input.ts

```diff
@@ -0,0 +1,127 @@
+// Bring in package dependencies
+import { Logger } from 'log4js';
```

Comment:
We don't want a specific log4js here. I believe we abstracted this in the package so that we can pass in any logger we want. Let's remove the specific reference to log4js and instead use the same interface we used in other places.

Reply:
Thats right, I had missed that. This may be an issue in the http input too...we can handle that as part of another PR though